static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -36,12 +39,14 @@ type SpanEvidenceKeyValueListProps = {
   causeSpans: Span[];
   event: EventTransaction;
   offendingSpans: Span[];
+  orgSlug: string;
   parentSpan: Span | null;
 };
 
 const TEST_ID_NAMESPACE = 'span-evidence-key-value-list';
 
 export function SpanEvidenceKeyValueList({event}: {event: EventTransaction}) {
+  const {slug: orgSlug} = useOrganization();
```

Comment:
Just wanted to note this,
Tbh, idk what the best practice for this is, in a lot of component we pass the organization prop around from the parent.

Reply:
We discussed this offline - going to leave as is for now.